go.mod

```diff
@@ -85,7 +85,6 @@ replace (
 	// dependency on itself through plugins. see .empty-mod/go.mod for details
 	github.com/containerd/containerd => ./.empty-mod/
 	// Use the relative local source of the github.com/containerd/containerd/api to build
```

Comment:
I realise this is likely not ready for merge, but in case I forget; this comment should be removed as well